[BUG] Automatic templates not working in DND5e V5.x and Foundry V13.x

**Module Version:** v0.7.0
**Foundry Version** V13.344
**Game System:** DND5e V5.0.3

Error: Failed to place measured template.. Cannot read properties of undefined (reading 'base')
[Detected 3 packages: tokenmagic(0.7.0), lib-wrapper(1.13.2.0), system:dnd5e(5.0.3)]
    at Hooks.onError (foundry.mjs:23678:24)
    at 🎁Hooks.onError#0 (libWrapper-wrapper.js:188:54)
    at #placeTemplate (mixin.mjs:1172:15)
    at SaveActivity._finalizeUsage (mixin.mjs:900:86)
    at SaveActivity.use (mixin.mjs:295:22)Caused by: TypeError: Cannot read properties of undefined (reading 'base')
[Detected 3 packages: tokenmagic(0.7.0), lib-wrapper(1.13.2.0), system:dnd5e(5.0.3)]
    at dnd5e.js:168:43
    at AbilityTemplate.eq (dnd5e.js:184:1)
    at 🎁call_wrapper [as call_wrapper] (libWrapper-wrapper.js:639:16)
    at 🎁game.dnd5e.canvas.AbilityTemplate.fromActivity#0 (libWrapper-wrapper.js:193:20)
    at #placeTemplate (mixin.mjs:1167:49)
    at SaveActivity._finalizeUsage (mixin.mjs:900:86)
    at SaveActivity.use (mixin.mjs:295:22)

**Describe the bug**
When automated templates are active Foundry throws the above error and no template appears to place on the map.

**To Reproduce**
Steps to reproduce the behavior:
1. Cast a spell with a template to place E.g. Fireball

**Expected behavior**
Template appears and has the Token Magic automatic template effect applied to it.

**Browser:**
 Chrome

--- a/tokenmagic/module/autoTemplate/dnd5e.js
+++ b/tokenmagic/module/autoTemplate/dnd5e.js
@@ -163,9 +163,20 @@ function fromCategories(categories = {}, activity, template) {
 
 	let config, dmgSettings;
 
+	let dmgTypes = [];
+	if(item.system?.damage?.base?.types) {
+		dmgTypes = item.system.damage.base.types;
+	} 
+	if(!dmgTypes.length && item.system.activities) {
+		const saves = item.system.activities.getByType("save")
+		if(saves.length) {
+			dmgTypes = saves.flatMap(save => save.damage.parts.flatMap(part => Array.from(part.types)))
+		}
+	}
+
 	// some items/spells have multiple damage types
 	// this loop looks over all the types until it finds one with a valid fx preset
-	for (const dmgType of item.system.damage.base.types) {
+	for (const dmgType of dmgTypes) {
 		dmgSettings = categories[dmgType] || {};
 		config = dmgSettings[template.t];
 

